nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/KeyStoreUtils.java

```diff
@@ -401,6 +401,7 @@ public static boolean isStoreValid(final URL keystore, final KeystoreType keysto
             return true;
 
         } catch (Exception e) {
+            logger.debug("Keystore could not be loaded", e);
```

Comment:
Recommend including the keystore location and keystore type for additional debugging:
```suggestion
            logger.debug("Keystore [{}] Type [{}] load failed", keystore, keystoreType, e);
```